docs/configure

```diff
@@ -87,6 +87,20 @@ hostlist   relay_from_hosts = localhost : @ : 192.168.0.0/24
 # Comment this line if you want to disable it, instead of + you can use a different separator.
 VEXIM_LOCALPART_SUFFIX = +*
 
+# If exim in your distro supports new 'headers_remove' behavior, define macro
+# below. See https://github.com/vexim/vexim2/pull/102 .
+# Debian 7	      4.80    old
+# Debian 8	      4.84    new
+# Debian 9	      4.86    new (not tested)
+# Ubuntu 12.04        4.76    old (not tested)
+# Ubuntu 14.04        4.82    old (not tested)
+# Ubuntu 16.04        4.86    new (not tested)
+# RHEL 6*             4.72    old (not tested)
+# RHEL 7*             4.84    new (not tested)
+# OpenSUSE 13.x       4.85    new (not tested)
+# OpenSUSE tumbleweed 4.85    new (not tested)
+#NEW_HEADERS_REMOVE = yes
```

Comment:
I think I'd name the macro `OLD_HEADERS_REMOVE` instead. And new syntax should be the default.